pkg/cloudprovider/fake/nodegroup.go

```diff
@@ -19,6 +19,8 @@ import (
 
 type NodeGroup struct {
 	WantErr  error
+	Stable   bool
```

Comment:
nit: can you call this stabilized?

Reply:
I had it as `Stabilized` but the method name conflicts with it.

Reply:
Hmm -- should we rename the method to IsStablized?

Reply:
It would have made sense to make this change, but, since this is all for testing/fake object `stable` field does the job, IMO.